src/api/job.ts

```diff
@@ -245,9 +234,9 @@ job.patch('/:jobId', async (req: Request, res: Response) => {
     res.send(response);
 });
 
-job.post('/:jobId', (req: Request, res: Response) => {
-    res.status(501).send();
-});
+// job.post('/:jobId', (req: Request, res: Response) => {
+//     res.status(501).send();
+// });
```

Comment:
Can you put a comment here saying why it's commented out, as well as a note in the spec for this endpoint saying that it's not implemented?